lua/telescope/_extensions/file_browser/picker.lua

```diff
@@ -25,6 +25,31 @@ local os_sep = Path.path.sep
 -- enclose in module for docgen
 local fb_picker = {}
 
+-- try to get the index of entry of current buffer
+local get_selection_index = function(opts, results)
+  local init_dir = Path:new(opts.path):absolute()
+  local buf_path = Path:new(vim.api.nvim_buf_get_name(0)):absolute()
```

Comment:
`vim.api.nvim_buf_get_name` gets the full file name of the buffer which should correspond to `Path:new(...):absolute()`.